pulseaudio-control.bash

```diff
@@ -49,7 +49,7 @@ function getCurVol() {
 # Saves the name of the sink passed by parameter into a variable named
 # `sinkName`.
 function getSinkName() {
-    sinkName=$(pactl list sinks short | awk -v sink="$1" '{ if ($1 == sink) {print $2} }')
+    sinkName=$(pactl list sinks | awk -v sink="Sink #$1" '$0 ~ sink {flag=1; next} flag && /Name:/{printf $2"/"} flag && /Active Port:/{print $NF; flag=0;}')
```

Comment:
What about:

```sh
sinkName=$(pactl list sinks | awk -v sink="Sink #$1" '$0 == sink {flag=1; next} flag && /Name:/{printf $2"/"; next} flag && /Active Port:/{print $NF; flag=0;}')
```

This avoids an unnecessary regular expression, and skips with `next` in the "Name" section, too.

Wouldn't this get messed up if a sink didn't have the "Name" parameter or "Active Port"? Or is that not going to happen?

Could you also add a comment above to explain this `awk` a bit?

Reply:
Removed the regex. Sinks should definitely have a `Name` parameter. The name comes up in `pactl list sinks short` too. I was under the impression that sinks should have an active port too. But in the tests here, the sinks created don't have one. I see that the sinks details are separated by an empty line in `pactl list sinks`. So, I have modified the awk command accordingly.
If a sink doesn't have an active port, its nickname will just be its name with a trailing /: `<name>/`